control/autoware_simple_pure_pursuit/src/simple_pure_pursuit.cpp

```diff
@@ -83,6 +84,7 @@ autoware_control_msgs::msg::Control SimplePurePursuitNode::create_control_comman
 
   // calculate control command
   autoware_control_msgs::msg::Control control_command;
+  control_command.stamp = get_clock()->now();
```

Comment:
In general, it’s better to propagate timestamp.

```suggestion
  control_command.stamp = odom.header.stamp;
```